ansible/playbooks/roles/repository/files/download-requirements/src/mode/red_hat_family_mode.py

```diff
@@ -98,14 +98,16 @@ def _add_third_party_repositories(self):
             self._tools.dnf_config_manager.add_repo('https://download.docker.com/linux/centos/docker-ce.repo')
             self._tools.dnf.accept_keys()
 
-        for repo in ['https://dl.2ndquadrant.com/default/release/get/10/rpm',  # for repmgr
-                     'https://dl.2ndquadrant.com/default/release/get/13/rpm']:
-            Command('curl', self._cfg.retries, [repo]) | Command('bash', self._cfg.retries)  # curl {repo} | bash
-
-        # script adds 2 repositories, only 1 is required
-        for repo in ['2ndquadrant-dl-default-release-pg10-debug',
-                     '2ndquadrant-dl-default-release-pg13-debug']:
-            self._tools.dnf_config_manager.disable_repo(repo)
+        # repmgr is supported only with x86_64 architecture
+        if 'x86_64' in self.__archs:
```

Comment:
We don't want to use magic consts.

Please change this conditional in such a way:
if self._cfg.os_arch == OSArch.X86_64

Don't forget to import OSArch type on top of the file.
from src.config.os_type import OSArch